### ISSUE TITLE
TypeError when clicking on link to LoadedLateAsClass

### ISSUE THREAD

**Issue by stenius** [iammerrick/how-to-lazy-load-react-webpack#4]
Hi, I'm getting an error when clicking the link in chrome and firefox after running npm install and webpack from your config files.

`Uncaught TypeError: Cannot read property 'hello' of null`

Can you please help me with this?  I'm having trouble figuring this out for a project I'm working on.

**Comment by iammerrick** [comment]
Thanks for reporting @stenius! It looks like the issue was due to how React handles callback refs. (They pass null when unmounting since it was a new function reference.) Rather than handle the null pass and a calling the method again I decided to drop the "ref" example because I think it adds unnecessary confusion to this example.

See [this issue](https://github.com/facebook/react/issues/4533) for more information.

Anyhow, I updated all the dependencies & pushed up a fix.
